src/main/webapp/data/userMapData.json

```diff
@@ -1328,6 +1364,7 @@
       "Northwood University",
       "Norwich Technical High School",
       "Nyack High School",
+      "NYU Stern",
```

Comment:
This seems to be a business school within `New York University`, which is already in the list.